<?xml version="1.0"?>

<launch>

  <node pkg="jaco2_calibration_utils" type="jaco2_static_data_generartion_node" name="static_data_generator_node" output="screen">
    <remap from="/joint_states" to="/jaco_arm_driver/out/joint_states"/>
    <param name="/driver_name" value="jaco_arm_driver"/>
    <param name="joint_1_upper_limit" value="5.108"/>
    <param name="joint_1_lower_limit" value="4.508"/>
    <param name="joint_2_upper_limit" value="3.26"/>
    <param name="joint_2_lower_limit" value="2.66"/>
    <param name="joint_3_upper_limit" value="1.30"/>
    <param name="joint_3_lower_limit" value="1.30"/>
    <param name="joint_4_upper_limit" value="4.50"/>
    <param name="joint_4_lower_limit" value="3.90"/>
    <param name="joint_5_upper_limit" value="1.75"/>
    <param name="joint_5_lower_limit" value="1.15"/>
    <param name="joint_6_upper_limit" value="1.62"/>
    <param name="joint_6_lower_limit" value="1.02"/>
    </node>
</launch>

